### PR TITLE
About page changes

### PR DESCRIPTION
**Pull request by verndrade**
Make about page visible to non-logged in users
Add testimonial to about page 
Edited some text
Changed github links to link to organization repository

### CODE REVIEW

File: templates/about.html [hunk 1]
```diff
@@ -1,66 +1,80 @@
...
+        </div>
+        <div class="col my-2 text-center profile">
+          <img src="{% static 'unknown.png' %}" class="rounded-circle mb-2">
+          <h3><a class="text-body" href=''>You!</a></h3>
+          <h4>UVA '2?</h4>
+          <h4>Bug Creator</h4>
+        </div>
+      </div>
+      <h2>Testimonials</h2>
+      <p>"I think it would benefit our students and potential students if more faculty members participated in this. I know that many student and friend groups keep copies of all syllabi and share them amongst themselves. I think posting them publicly on a site like this helps level the playing field for students who do not belong to those groups. It also seems like this would make my life easier, having students get my syllabi without dealing with all the personal requests through email. I'm glad you are working on this." ~ Prof. Raymond Pettit, UVA Computer Science</p>
```
Comment (winstliu) [inline templates/about.html:69]: Just making sure that we do have consent to display this publicly?

**Review by winstliu — COMMENTED** [review]
Just one concern

File: syllabiShare/views.py [hunk 2]
```diff
@@ -8,11 +8,6 @@
 
 
 def about(request):
-    (template, context) = authenticate(request.user)
```
Comment (winstliu) [inline syllabiShare/views.py:11]: Oh wait, and what will the user menu dropdown show if they're logged out

File: templates/about.html (hunk 1 above)
Comment (verndrade) [reply templates/about.html:69]: Yes I asked Pettit for permission!

File: syllabiShare/views.py (hunk 2 above)
Comment (verndrade) [reply syllabiShare/views.py:11]: If you look at the page I removed the nav bar

**Review by mrf7pc — APPROVED** [review]

**Pull request merged**